config/routes.rb

```diff
@@ -1,10 +1,30 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  root 'admin_users#index'
+  
+  resources :admin_users do
+    collection do
+      get :login
+      post :attempt_login
+      delete :logout 
+    end
+  end 
+
```

Comment:
remove extra line here and also to the following cases

Reply:
removed from every file